DER/DerivativesContracts/ExoticOptions.rdf

```diff
@@ -591,7 +583,7 @@ Further notes: REVIEW THIS - there appear to be two separate semantics in this l
 	<owl:ObjectProperty rdf:about="&fibo-der-der-exo;enteredIntoBy">
 		<rdfs:label xml:lang="en">entered into by</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-der-der-exo;OptionPayoutCommitment"/>
-		<rdfs:range rdf:resource="&fibo-der-der-opt;OptionIssuer"/>
+		<rdfs:range rdf:resource="&fibo-der-drc-opt;OptionIssuer"/>
```

Comment:
@rivettp Ok - see above - when we get to this ontology it can go away, but will need to make sure it is not used elsewhere, which some properties are.